frontend/app/views/spree/shared/_option_values.html.erb

```diff
@@ -1,4 +1,4 @@
-<% cache base_cache_key + [ot_downcase_name, params_ot_downcase_name, filtering_params_cache_key, @taxon&.id] do %>
+<% cache option_type_cache_key(option_type.filter_param) do %>
```

Comment:
```suggestion
<% cache option_type_filter_cache_key(option_type) do %>
```

let's pass `option_type` object, this will be more future-proof and more flexible, at the same time less logic in views